pom.xml

```diff
@@ -35,6 +35,7 @@
 	
 	<properties>
 		<openMRSVersion>1.9.9</openMRSVersion>
+		<legacyuiVersion>1.7.0-SNAPSHOT</legacyuiVersion>
```

Comment:
Could there be a reason why you are depending on the snapshot version of the module?

Reply:
Thanks @dkayiwa ! No, I was just testing against my local Rwanda environment. I will update this dependency to the latest legacyui release. Thanks.